content/guides/embed/ui-elements/theming-styling.md

```diff
@@ -0,0 +1,122 @@
+---
+rank: 11
+related_endpoints: []
+related_guides:
+  - embed/ui-elements
+  - embed/ui-elements/explorer
+  - embed/ui-elements/uploader
+required_guides:
+  - embed/ui-elements/installation
+related_resources: []
+---
+<!--alex ignore -->
+# Theming & styling for Box UI Elements
+
+With theming & styling for Box UI Elements you can customize the look of the
+embedded Box component to your enterprise's requirements.
+<!--alex ignore -->
+The tool provides a real-time live preview for an easy adjustment of the theme.
+<!--alex enable -->
+
+<!--alex ignore -->
+<Message type='notice'>As of now, theming & styling is available for Content
+Explorer and Content Uploader.
+</Message>
+<!--alex enable -->
+
+## How to start
+
+Add the selected BUIE to your application. The default theme template will be applied.
+
+## Customization
+
+Box uses design tokens for the customization purposes.
+
+<Message type='notice'>
+Design tokens are named entities that store specific visual attributes, such
+as colors, spacing, typography, scale, and many more.
+They are used instead of hard-coded values to simplify sharing design
+properties across platforms, tools, and components.
+</Message>
+
+You can change the following elements in your embedded Box component.
+<!--alex ignore -->
+
+### Colors
+
+Customize:
+
+<!--alex ignore -->
+* global color palette - select the primary, secondary, and access colors
+* state colors - select color for states such as hover, focus, active, disabled
+* component-specific colors - for example, you can select different colors for your buttons depending on their type
+<!--alex enable -->
+* gradients - choose and adjust a gradient as your background or other components
+* opacity - adjust the opacity of the overlay effects for Box Modals and side panels
+
+### Typography
+
+Customize: 
+
+* global typography settings - choose the font family, size, and weights
```

Comment:
This is not working right now, as Trevor mentioned. Not sure if we should add this